t-circulating-supply/main.py

```diff
@@ -27,7 +27,7 @@
 #  will equal total supply
 NU_CIRCULATING_SUPPLY_PRE_NUCO_VESTING_DATE = 1_180_688_920.6442547  # original NU supply endpoint returns this value before October 15, 2025
 NU_FINAL_CIRCULATING_SUPPLY_POST_NUCO_VESTING_DATE = 1_380_688_920.6442547  # difference of 200M NU (equals supply of NU used for Threshold merger)
-NUCO_VESTING_DATE = maya.MayaDT.from_rfc3339('2025-10-15T00:00:00.0Z')
+NUCO_VESTING_DATE = datetime.datetime(2025, 10, 15, 0, 0, 0)
```

Comment:
We have to be careful. The vesting date is really midnight UTC time.

```suggestion
NUCO_VESTING_DATE = datetime.datetime(2025, 10, 15, 0, 0, 0, tzinfo=datetime.timezone.utc)
```

Reply:
384133ad48010def05116ded4e8b18fc159f592f